src/rust/madara/sources/prismascans/src/lib.rs

```diff
@@ -0,0 +1,47 @@
+#![no_std]
+use aidoku::{
+	error::Result, prelude::*, std::String, std::Vec, Chapter, DeepLink, Filter, Listing, Manga,
+	MangaPageResult, Page,
+};
+
+use madara_template::template;
+
+fn get_data() -> template::MadaraSiteData {
+	let data: template::MadaraSiteData = template::MadaraSiteData {
+		base_url: String::from("https://prismascans.net"),
+		description_selector: String::from("div.summary_content_wrap div p"),
+		alt_ajax: true,
+		..Default::default()
+	};
+	data
```

Comment:
should just be able to do this
```suggestion
	template::MadaraSiteData {
		base_url: String::from("https://prismascans.net"),
		description_selector: String::from("div.summary_content_wrap div p"),
		alt_ajax: true,
		..Default::default()
	}
```